Tools/machines/tuolumne-llnl/install_cpu_dependencies.sh

```diff
@@ -0,0 +1,240 @@
+    -DBUILD_TESTING=OFF          \
+    -DHDF5_ENABLE_PARALLEL=ON    \
+    -DCMAKE_INSTALL_PREFIX=${SW_DIR}/hdf5-1.14.1.2
+cmake \
+    --build ${build_dir}/hdf5-build \
+    --target install                \
+    --parallel ${build_procs}
+rm -rf ${build_dir}/hdf5-build
+
+# ADIOS2
+if [ -d ${SRC_DIR}/adios2 ]
+then
+  cd ${SRC_DIR}/adios2
+  git fetch --prune
+  git checkout v2.10.2
+  cd -
+else
+  git clone -b v2.10.2 https://github.com/ornladios/ADIOS2.git ${SRC_DIR}/adios2
+fi
+cmake \
+    --fresh                      \
+    -S ${SRC_DIR}/adios2         \
+    -B ${build_dir}/adios2-build \
+    -DADIOS2_USE_Blosc2=ON       \
+    -DADIOS2_USE_Campaign=OFF    \
+    -DADIOS2_USE_Fortran=OFF     \
+    -DADIOS2_USE_Python=OFF      \
+    -DADIOS2_USE_ZeroMQ=OFF      \
+    -DBUILD_SHARED_LIBS=OFF      \
+    -DCMAKE_INSTALL_PREFIX=${SW_DIR}/adios2-2.10.2
+cmake \
+    --build ${build_dir}/adios2-build \
+    --target install                  \
+    --parallel ${build_procs}
+rm -rf ${build_dir}/adios2-build
+
+# BLAS++ (for PSATD+RZ)
+if [ -d ${SRC_DIR}/blaspp ]
+then
+  cd ${SRC_DIR}/blaspp
+  git fetch --prune
+  git checkout v2024.05.31
+  cd -
+else
+  git clone -b v2024.05.31 https://github.com/icl-utk-edu/blaspp.git ${SRC_DIR}/blaspp
+fi
+cmake \
+    --fresh \
+    -S ${SRC_DIR}/blaspp                      \
+    -B ${build_dir}/blaspp-tuolumne-cpu-build \
+    -Duse_openmp=ON                           \
+    -Dgpu_backend=OFF                         \
+    -DBUILD_SHARED_LIBS=OFF                   \
+    -DCMAKE_CXX_STANDARD=17                   \
+    -DCMAKE_INSTALL_PREFIX=${SW_DIR}/blaspp-2024.05.31
+cmake \
+    --build ${build_dir}/blaspp-tuolumne-cpu-build \
+    --target install                               \
+    --parallel ${build_procs}
+rm -rf ${build_dir}/blaspp-tuolumne-cpu-build
+
+# LAPACK++ (for PSATD+RZ)
+if [ -d ${SRC_DIR}/lapackpp ]
+then
+  cd ${SRC_DIR}/lapackpp
+  git fetch --prune
+  git checkout v2024.05.31
+  cd -
+else
+  git clone -b v2024.05.31 https://github.com/icl-utk-edu/lapackpp.git ${SRC_DIR}/lapackpp
+fi
+cmake \
+    --fresh                                     \
+    -S ${SRC_DIR}/lapackpp                      \
+    -B ${build_dir}/lapackpp-tuolumne-cpu-build \
+    -DCMAKE_CXX_STANDARD=17                     \
+    -Dgpu_backend=OFF                           \
+    -Dbuild_tests=OFF                           \
+    -DBUILD_SHARED_LIBS=OFF                     \
+    -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON      \
+    -DCMAKE_INSTALL_PREFIX=${SW_DIR}/lapackpp-2024.05.31
+cmake \
+    --build ${build_dir}/lapackpp-tuolumne-cpu-build \
+    --target install                                 \
+    --parallel ${build_procs}
+rm -rf ${build_dir}/lapackpp-tuolumne-cpu-build
+
+# PETSC
+if [ -d ${SRC_DIR}/petsc ]
+then
+  cd ${SRC_DIR}/petsc
+  git fetch --prune
+  git checkout v3.24.0
+  cd -
+else
+  git clone -b v3.24.0 https://gitlab.com/petsc/petsc.git ${SRC_DIR}/petsc
+fi
+cd petsc
```

Comment:
Forgot the
```suggestion
cd ${SRC_DIR}/petsc
```
Fixed via #6410